Failed to run backend development server

Backend server was failing to run with the command "npm run dev"
while frontend server was fine.

It was showing an error: "npm ERR! missing script: server"

![issue](https://user-images.githubusercontent.com/66744273/101975348-e8ca5300-3c61-11eb-90e9-cdd3241defe8.JPG)

--- a/package.json
+++ b/package.json
@@ -5,9 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "nodemon server.js",
-    "server": "nodemon server",
     "ui": "cd ui && npm start",
-    "dev": "concurrently \"npm run server\" \"npm run ui\""
+    "dev": "concurrently \"npm run start\" \"npm run ui\""
   },
   "proxy": "http://localhost:5000",
   "repository": {

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "main": "server.js",
   "scripts": {
     "start": "nodemon server.js",
+    "server": "nodemon server",
     "ui": "cd ui && npm start",
     "dev": "concurrently \"npm run server\" \"npm run ui\""
   },

